TundraTD/Assets/Scripts/SceneManagement/EndLevelMenuScript.cs

```diff
@@ -0,0 +1,25 @@
+﻿using UnityEngine;
+using UnityEngine.SceneManagement;
+
+namespace SceneManagement
+{
+    public class EndLevelMenuScript : MonoBehaviour
+    {
+        private const int LevelsSceneID = 0;
+        
+        public void KeepPlaying(string result)
```

Comment:
The same thing as in previous review. I guess it should be changed because strings are quiet hard to compute so using enums will increase potential performance of the game

Reply:
Ok, I see there's no way to pass enums in editor so it's a Unity fault. Well then, let it be as it was